test/spec/modules/pubmaticBidAdapter_spec.js

```diff
@@ -124,6 +126,59 @@ describe('PubMatic adapter', function () {
       }
     ];
 
+    nativeBidRequests = [{
+      code: '/19968336/prebid_native_example_1',
+      sizes: [
+        [300, 250]
+      ],
+      mediaTypes: {
+        native: {
+          title: {
+            required: true,
+            length: 50
+          },
+          image: {
+            required: true,
```

Comment:
need a case where required is set to false or is not present

Reply:
done